docs/pages/guides/simple-scene.md

```diff
@@ -0,0 +1,448 @@
+  </TabItem>
+</Tabs>
+
+### Register output stream `output_1`.
+
+Configure it to render just an empty [`View`](../api/components/View.md) component with a background color set to `#4d4d4d` (gray).
+
+<Tabs queryString="lang">
+  <TabItem value="http" label="HTTP">
+    ```http
+    POST: /api/output/output_1/register
+    Content-Type: application/json
+
+    {
+      "type": "rtp_stream",
+      "transport_protocol": "tcp_server",
+      "port": 9003,
+      "video": {
+        "resolution": { "width": 1280, "height": 720 },
+        "encoder": {
+          "type": "ffmpeg_h264",
+          "preset": "ultrafast",
+          "initial": {
+            "root": {
+              "type": "view",
+              "background_color_rgba": "#4d4d4dff",
+            }
+          }
+        }
+      }
+    }
+    ```
+
+    After receiving the response you can establish the connection and start listening for the stream. Checkout [how to receive output streams](./receive-output.md) to learn more.
+
+    In this example we are using RTP over TCP, if you prefer to use UDP you need start listening on the specified port before sending register request to make sure you are not losing
+    first frames.
+  </TabItem>
+  <TabItem value="membrane" label="Membrane Framework">
+    ```elixir
+    def handle_init(ctx, opts) do
+      spec = [
+        ...
+
+        get_child(:live_compositor),
+        |> via_out(Pad.ref(:video_output, "output_1"), options: [
+          width: 1280,
+          height: 720,
+          encoder: %Encoder.FFmpegH264{
+            preset: :ultrafast
+          },
+          initial: %{
+            root: %{
+              type: :view,
+              background_color_rgba: "#4d4d4dff",
+            },
+          },
+        ])
+        |> output_1_spec
+
+        ...
+      ]
+      {[spec: spec], %{}}
+    end
+    ```
+    where `output_1_spec` is an element that can consume H264 video.
+  </TabItem>
+</Tabs>
+
+`View` component is empty, so on the output you should see just a blank screen of a specified color as shown below.
+
+<div style={{textAlign: 'center'}}>
+    <img src={SimpleSceneEmpty} style={{ width: 600 }} />
+    Output stream
+</div>
+
+## Update scene to show an input
+
+Update output to render a [`View`](../api/components/View.md) component with an [`InputStream`](../api/components/InputStream.md) as its child.
+
+<Tabs queryString="lang">
+  <TabItem value="http" label="HTTP">
+    ```http
+    POST: /api/output/output_1/update
+    Content-Type: application/json
+
+    {
+      "video": {
+        "root": {
+          "type": "view",
+          "background_color_rgba": "#4d4d4dff",
+          "children": [
+            { "type": "input_stream", "input_id": "input_1" },
+          ]
+        }
+      }
+    }
+    ```
+  </TabItem>
+  <TabItem value="membrane" label="Membrane Framework">
+    ```elixir
+    def handle_setup(ctx, state) do
+      request = %LiveCompositor.Request.UpdateVideoOutput{
```

Comment:
alisa Membrane.LiveCompositor

Reply:
I'll add alias in the first snippet but skip it everywhere else